eng/pipelines/templates/VSCode-release.template.yml

```diff
@@ -1,15 +1,24 @@
 ---
+parameters:
+  rids: ["win-x64" , "osx-x64", "linux-x64"]
```

Comment:
>] [](http://example.com/codeflow?start=43&length=1)

linux-arm64? 
Did Alpine work before?
Do we want win10-arm64 and/or win7-x86?

Reply:
We have not shipped a version of Alpine before, but there is an ask for it. I'll include it including the linux-arm64, win10-arm64, and win7-x86.

We do not ship an osx-arm64 ATM since there is no native lldb-mi for osx-arm64.